message SearchRequest {
	optional int32 req = 1;
}

message SearchResponse {
	optional int32 resp = 1;
}

service SearchService {
	rpc Search (SearchRequest) returns (SearchResponse);
}
